federatedscope/vertical_fl/xgb_base/worker/XGBClient.py

```diff
@@ -48,6 +48,13 @@ def __init__(self,
 
         self.data = data
         self.own_label = ('y' in self.data['train'])
+
+        self.test_x = self.data['test']['x']
+        if self.own_label:
+            self.test_y = self.data['test']['y']
+
+        self.test_z = np.zeros(self.test_x.shape[0])
```

Comment:
what `test_z` means?

Reply:
It is exactly the result outputted by the model on test data, I'll change the name.